GoogleForCreators/web-stories-wp
# Design System: Styles for DropDown

## Summary

Styles the new dropdown

## Relevant Technical Choices

- Following[ these figma designs](https://www.figma.com/file/bMhG3KyrJF8vIAODgmbeqT/Design-System?node-id=3397%3A800)
- Separating out styles into `components.js` to keep file size down. 
- Sets up disabled menu item handling 

## To-do

- [ ] Follow up on edge case sizing 
- [ ] Follow up on any viewport responsive min/max widths we want to set 

As separate PRs: 

- [ ] a11y pass on dropdown 
- [ ] Scrollbar styles 
- [ ] Preserve icon thickness 
- [ ] Implement focus only on keyboard user 


## User-facing changes

- None, exists only in storybook 

## Testing Instructions

- Spin up a local copy of storybook and navigate to designsystem/components/dropdown and look at variants. 

[Here is a copy of my outstanding questions for how this should look.](https://docs.google.com/document/d/1w06W9T3sxbiqveoz9Zk9_sO87JPO4seQzjo1FIDBK54/edit?usp=sharing)

Light theme: 
![Screen Shot 2020-12-22 at 3 09 17 PM](https://user-images.githubusercontent.com/10720454/102941256-7df1f680-446f-11eb-88bd-431ca12ea796.png)

Dark theme:
![Screen Shot 2020-12-22 at 3 09 47 PM](https://user-images.githubusercontent.com/10720454/102941257-7df1f680-446f-11eb-93cb-1319bf092918.png)

Nested: 
![Screen Shot 2020-12-22 at 3 10 08 PM](https://user-images.githubusercontent.com/10720454/102941255-7d596000-446f-11eb-8ef4-4104fd95572b.png)


---



Addresses #4950
